unwind/tests/correctness.rs

```diff
@@ -25,24 +25,32 @@ fn test_frame_3() {
         println!("{:08x}", i);
     }
     println!();
-        
-    DwarfUnwinder::default().trace(move |frames| {
+
+    let mut ref_trace_len = ref_trace.len();
+    // Haven't investigated why this has a trailing zero.
```

Comment:
I have this too in https://github.com/bjorn3/pretty_backtrace/, it seems a property of `libbacktrace`.